programmers/난이도별/level01.문자열_내_p와_y의_개수/develop-sell.js

```diff
@@ -0,0 +1,29 @@
+function solution(s){
+  var answer = true;
+  var ans = 0;
+  for(let i =0; i < s.length; i++){
+      let char = s.charCodeAt(i);
+      (char === 80 || char === 112) ? ans += 1 : '';
```

Comment:
```suggestion
      (char === 80 || char === 112) && (ans += 1);
```
이런식으로 실행도 가능할 것 같습니다!

Reply:
아! 앞의 식이 true면 뒤에 && 부분을 바로 진행하게 되는건가요? 
true가 아니면 뒤의 식은 무시하고 바로 내려가니까????